core/src/main/java/com/hcl/ecomm/core/services/CustomerService.java

```diff
@@ -12,6 +12,8 @@
 
 	public String customerProfileServicePath();
 
+	public String getRecaptchaSiteKey();
```

Comment:
@nitheshkumar-hcl  put this method in common service

Reply:
Committed the changes as per the review comments